fota - various fixes with the goal to set a proper result once fota aborts (#7479)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

* Updated:

  * :ref:`lib_lwm2m_client_utils` library:

    * Fixed:

>       * Set update result on FOTA correctly
>       * Report result on FOTA update back to lwm2m

Reviewer: back to the LwM2M server?
```suggestion
      * Setting of the FOTA update result.
      * Reporting of the FOTA update result back to the LwM2M server.
```
